Gemfile

```diff
@@ -1,19 +1,20 @@
 
 source 'https://rubygems.org'
 
-gem 'nanoc'
-gem 'kramdown'
-gem 'builder'
-gem 'sass'
-gem 'rake'
+gem 'nanoc'            # engine
+gem 'kramdown'         # markdown converter
+gem 'rouge'            # syntax highlighting
+gem 'slim'             # templates
+gem 'builder'          # xml sitemaps
+gem 'sass'             # css conversion
 
-group :development do
-  gem 'guard-nanoc'
-  gem 'adsf'
-  gem 'thin'
+group :nanoc do
+  gem 'guard-nanoc'    # automatic rebuilds
+  gem 'adsf'           # tiny webserver
+  gem 'puma'           # webserver backend
 end
 
 group :test do
-  gem 'nokogiri'
-  gem 'w3c_validators'
+  gem 'nokogiri'       # 
```

Comment:
nokogiri is the HTML parser in case you want to add a comment here. Love this library, use it for all my quick'n'dirty parsing tasks.